src/v/raft/consensus.h

```diff
@@ -331,8 +331,12 @@ class consensus {
     size_t received_snapshot_bytes() const { return _received_snapshot_bytes; }
     bool has_pending_flushes() const { return _has_pending_flushes; }
 
+    log_start_offset_delta last_snapshot_delta() const {
+        return _last_start_delta;
+    }
```

Comment:
this name is confusing, i proposed storing an absolute value for an effective start offset, this way we would get rid of delta we can have then `log_start_offset` and `start_offset` or something similar ?